pontus/week1/ass2/index.html

```diff
@@ -1,30 +1,160 @@
-</head>
-<body>
-
-	<div id="app"></div>
-
-	<!-- 1. Create an inputfield that doesn't accept any numbers -->
-	<!-- 2. Find a way to submit and print the message -->
-	<!-- 3. Validation errors in inputfields should produce an error-message rather than a hello morld message -->
-	<!-- 4. Any error-message should be printed in red color -->
-	<!-- 5. Provide a function that returns all messages ever printed -->
-	<!-- 6. This must be solved by using only native Javascript and react -->
-	<!-- 7. All markup must be produced with react -->
-	<!-- 8. All messages must be printed in the order they where submitted -->
-
-</body>
+	<head>
+		<title>Hello world with react</title>
+		<style type="text/css">
+body {
+	width: 1000px;
+	margin: auto;
+}
+		</style>
+
+		<script src="http://fb.me/react-0.14.7.min.js"></script>
+		<script src="http://fb.me/react-dom-0.14.7.min.js"></script>
+		<!-- <script src="http://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.4.4/babel.min.js"></script> -->
+		<script src="http://fb.me/JSXTransformer-0.12.2.js"></script>
+	</head>
+	<body>
+
+		<div id="app"></div>
+
+		<!-- 1. Create an inputfield that doesn't accept any numbers -->
+		<!-- 2. Find a way to submit and print the message -->
+		<!-- 3. Validation errors in inputfields should produce an error-message rather than a hello morld message -->
+		<!-- 4. Any error-message should be printed in red color -->
+		<!-- 5. Provide a function that returns all messages ever printed -->
+		<!-- 6. This must be solved by using only native Javascript and react -->
+		<!-- 7. All markup must be produced with react -->
+		<!-- 8. All messages must be printed in the order they where submitted -->
+
+
+
+		<script type="text/jsx" >
+var Wrapper = React.createClass({ 
+			getInitialState: function() { 
+
+			return { 
+					text: "",
+					messages : [],
+					error: false
+				} 
+
+			},
+
+			handleCorrectInput: function(val) { 
+
+				this.state.messages.push(val);
+
+				// update the state.
+				this.setState({ 
+					error: false,
+					text: val
+				});
+			},
+
+			handleIncorrectInput: function(error) { 
+
+				this.setState({ 
+					text:error, 
+					error: true
+				});
+
+
+			},
+
+			showAll: function() { 
+
+			var str = this.state.messages.join('\n');
+
+				this.setState({ 
+					text : str
+				});
+
+			},
+
+			render: function() { 
+
+
+				return ( 
+						<div>
+							<InputField handleIncorrectInput={this.handleIncorrectInput} handleCorrectInput={this.handleCorrectInput} text={this.state.text} />
+							<OutputField error={this.state.error} showAll={this.showAll} text={this.state.text} messages={this.state.messages} />
+						</div>
+				);
+			}
+		});
+
+
+
+
+		// input field
+
+		var InputField = React.createClass({ 
+
+			handleClick() { 
+
+				var re = /\d/;
+				var val = this.refs.inputField.value;
+
+				if (re.test(val)) { 
+					var error = "error: must use only alphabetic characters.";
+
+
+					// callback to method in wrapper
```

Comment:
The code should always be written clear enough so that it is able to document itself. Unless it is really needed (and I do not think this is the case _here_) then you should not write comments. 

The next person who edits this file might not care about updating the comment once he/she will have updated the corresponding code.